pyxdf/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='pyxdf',
+      version='0.1',
+      description='Python library for importing XDF (Extensible Data Format)',
+      url='http://github.com/sccn/xdf',
+      author='Chadwick Boulay',
+      author_email='chadwick.boulay@gmail.com',
```

Comment:
Should be `kothe@sccn.ucsd.edu` (or the email of Alejandro ... depending on who is listed as author)